ref/feature/samlWeb/examples.adoc

```diff
@@ -0,0 +1,33 @@
+**Examples**
+
+The `samlWebSso20` element is configured by default in Liberty.
+
+----
+<samlWebSso20 id="defaultSP">
+
+</samlWebSso20>
+----
+
+The following default values are assumed in this default configuration,
+
+•	AssertionConsumerService URL:
+
+`https://<hostname>:<sslport> /ibm/saml20/defaultSP/acs`
+
+•	Service Provider (SP) metadata URL:
+
+`https://<hostname>:<sslport> /ibm/saml20/defaultSP/samlmetadata`
+
+This URL is used to download the metadata for the service provider (SP) and provide the URL to the SAML identity provider to establish federation between this SP and Identity Provider (IdP).
```

Comment:
`This URL is used to download the metadata for the service provider (SP) and provide the URL to the SAML identity provider to establish federation between this SP and Identity Provider (IdP).`

- Is `This URL..` the same URL as `provide the URL to the SAML identity provider...` It's not clear
- is `the SAML identity provider` the same entity as `Identity Provider (IdP)` ? Is so, abbreviate on first mention and use the abbreviation on the second mention. If not, needs to be more clear what the difference is.
- can this sentence be in active voice? ie state _what_ is using the URL to dl metadata and make that the subject of the sentence

Reply:
I think SAML is the service provider here.